<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-A9GEUTV" failures="0" tests="8" name="TestNGExample.Example5InvocationCount" time="3.070" errors="1" timestamp="2024-04-18T07:33:04 IST" skipped="1">
  <testcase classname="TestNGExample.Example5InvocationCount" name="testCase1" time="0.011"/>
  <testcase classname="TestNGExample.Example5InvocationCount" name="testCase1" time="0.004"/>
  <testcase classname="TestNGExample.Example5InvocationCount" name="testCase1" time="0.008"/>
  <testcase classname="TestNGExample.Example5InvocationCount" name="testCase1" time="0.004"/>
  <testcase classname="TestNGExample.Example5InvocationCount" name="testCase1" time="0.001"/>
  <testcase classname="TestNGExample.Example5InvocationCount" name="testCase2" time="2.029">
    <error message="Method TestNGExample.Example5InvocationCount.testCase2() didn&#039;t finish within the time-out 2000" type="org.testng.internal.thread.ThreadTimeoutException">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method TestNGExample.Example5InvocationCount.testCase2() didn't finish within the time-out 2000
at java.base@17.0.6/java.lang.Thread.sleep(Native Method)
at app//TestNGExample.Example5InvocationCount.testCase2(Example5InvocationCount.java:16)
at java.base@17.0.6/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base@17.0.6/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base@17.0.6/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base@17.0.6/java.lang.reflect.Method.invoke(Method.java:568)
at app//org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at app//org.testng.internal.invokers.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at app//org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:76)
at app//org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:11)
at java.base@17.0.6/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base@17.0.6/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base@17.0.6/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base@17.0.6/java.lang.Thread.run(Thread.java:833)
]]>
    </error>
  </testcase> <!-- testCase2 -->
  <testcase classname="TestNGExample.Example5InvocationCount" name="testCase4" time="1.013"/>
  <testcase classname="TestNGExample.Example5InvocationCount" name="testCase3">
    <skipped/>
  </testcase> <!-- testCase3 -->
</testsuite> <!-- TestNGExample.Example5InvocationCount -->
